runbotics/runbotics-orchestrator-ui/src/src-app/components/Accordion/Accordion.tsx

```diff
@@ -5,16 +5,16 @@ import { Box, Typography } from '@mui/material';
 import { CustomDivider, StyledAccordion, StyledAccordionDetails, StyledAccordionSummary, StyledExpandMoreOutlinedIcon } from './Accordion.styles';
 import { AccordionProps } from './Accordion.types';
 
-const Accordion: FC<AccordionProps> = ({ title, children }) => {
-    const [isExpanded, setIsExpanded] = useState(false);
+const Accordion: FC<AccordionProps> = ({ title, children, defaultExpanded = false, props }) => {
+    const [isExpanded, setIsExpanded] = useState(defaultExpanded);
     const [isHovered, setIsHovered] = useState(false);
 
     const toggleExpand = () => (_, isOpen) => {
         setIsExpanded(isOpen);
     };
 
     return (
-        <StyledAccordion $expanded={isExpanded} onChange={toggleExpand()}>
+        <StyledAccordion $expanded={isExpanded} onChange={toggleExpand()} expanded={isExpanded} {...props}>
```

Comment:
Why do we need expanded props twice? First with $, second without?

Reply:
one with $ is for styled components, the other one is for react logic